src/client/mod.rs

```diff
@@ -91,31 +92,30 @@ impl Client {
     /// # Examples
     ///
     /// Create a random client
-    /// ```
-    /// # extern crate tokio;
-    /// # use safe_core::CoreError;
+    /// ```no_run
+    /// # extern crate tokio; use safe_core::CoreError;
     /// use safe_core::Client;
-    /// use std::str::FromStr;
     ///
-    /// # #[tokio::main]
-    /// # async fn main() {
-    /// # let _: Result<(), CoreError> = futures::executor::block_on( async {
+    /// # #[tokio::main] async fn main() { let _: Result<(), CoreError> = futures::executor::block_on( async {
     ///
     /// let mut client = Client::new(None).await?;
     /// // Now for example you can perform read operations:
-    /// let some_balance = client.get_balance().await?;
-    ///
-    /// # Ok(())
-    /// # } );
-    /// # }
-    ///
+    /// let _some_balance = client.get_balance().await?;
+    /// # Ok(()) } ); }
     /// ```
     pub async fn new(sk: Option<SecretKey>) -> Result<Self, CoreError> {
         crate::utils::init_log();
+
+        #[cfg(feature = "simulated-payouts")]
         let mut is_random_client = true;
```

Comment:
```suggestion
        let mut is_random_client = sk.is_none();
```
This will let us remove lines 114-118